.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2025 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Create a release
+
+on:
+  workflow_dispatch:
+    inputs:
+      rel_version:
+        description: 'Release version (examples: 1.16.0-rc.1, 1.16.0)'
+        required: true 
+        type: string
+
+permissions: {}
+
+jobs:
+  create-release:
```

Comment:
Could we consider gating the workflow dispatch to maintainers and the release team?

https://github.com/tspascoal/get-user-teams-membership Here's an action that I would consider reasonable to implement, wdyt?

Reply:
I think that already happens today, only certain people can run workflows